docs/server_configuration.rst

```diff
@@ -241,6 +246,21 @@ Example server configuration
      - Name prefix for threads created by vector merge threadpool executor
      - VectorMergeExecutor
 
+   * - commit.maxThreads
+     - int
+     - Size of commit threadpool executor
+     - 5
```

Comment:
Should the default number be 1 since only one commit can happen at a time for an index?

Reply:
The synchronized scope for commit is reduced in v1 https://github.com/Yelp/nrtsearch/blob/main/src/main/java/com/yelp/nrtsearch/server/index/ShardState.java#L358 this allows multiple pending commits to join the same upload task. So it is good for them to run concurrently.